Makefile

```diff
@@ -84,6 +84,10 @@ FCBINDIR      := $(PREFIXDEPS)/bin
 SYSCONFDIR    := /etc
 LOCALSTATEDIR := /var
 
+# default CMD names if not
+QEMUCMD       ?= kata-qemu-lite-system-x86_64
```

Comment:
What's this? We don't use a kata prefix as this is not a Kata-developed component.